tests/test_site/settings.py

```diff
@@ -74,8 +74,14 @@
 
 # OpenTracing settings
 
-OPENTRACING_TRACE_ALL = True
-OPENTRACING_TRACER = django_opentracing.DjangoTracer(opentracing.Tracer())
-OPENTRACING_TRACED_ATTRIBUTES = ['META', 'FAKE_ATTRIBUTE']
+import appdash
```

Comment:
I'd rather not assume that users are running a particular OpenTracing implementation (or at least I'd rather not assume it in this way).

An alternative would be to have the developer somehow override the `OPENTRACING_TRACER`.